app/src/main/java/com/github/se/wanderpals/model/viewmodel/AgendaViewModel.kt

```diff
@@ -43,14 +43,48 @@ open class AgendaViewModel(
 
   open var selectedDate: LocalDate? = LocalDate.now()
 
+  open val _stopsInfo = MutableStateFlow<Map<LocalDate, CalendarUiState.StopStatus>>(emptyMap())
```

Comment:
Looks good. It may be simpler to use a simple Boolean instead of a custom enum but if you have ideas of other states the stop could be in, it can be a good solution.